js/checkbox/Example.svelte

```diff
@@ -9,7 +9,7 @@
 	class:gallery={type === "gallery"}
 	class:selected
 >
-	{value.toLocaleString()}
+	{value ? value.toLocaleString() : ""}
```

Comment:
Thanks for pointing this out @hannahblair. As I was testing this, I realized that my original approach won't work since `value` for `Checkbox` can be a boolean and if its False, that gets rendered as an empty string. Will revise